src/o365/spo/commands/orgassetslibrary/orgassetslibrary-list.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  ContextInfo, ClientSvcResponse, ClientSvcResponseContents
+} from '../../spo';
+import config from '../../../../config';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+import { OrgAssetsResponse, OrgAssets } from './OrgAssets';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+}
+
+class SpoOrgNewsSiteListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.ORGASSETSLIBRARY_LIST}`;
+  }
+
+  public get description(): string {
+    return 'List all libraries that are assigned as asset library';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let spoAdminUrl: string = '';
+
+    this
+      .getSpoAdminUrl(cmd, this.debug)
+      .then((_spoAdminUrl: string): Promise<ContextInfo> => {
+        spoAdminUrl = _spoAdminUrl;
+
+        return this.getRequestDigest(spoAdminUrl);
+      })
+      .then((res: ContextInfo): Promise<string> => {
+        const requestOptions: any = {
+          url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
+          headers: {
+            'X-RequestDigest': res.FormDigestValue
+          },
+          body: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="4" ObjectPathId="3" /><Query Id="5" ObjectPathId="3"><Query SelectAllProperties="true"><Properties /></Query></Query><Method Name="GetOrgAssets" Id="6" ObjectPathId="3" /></Actions><ObjectPaths><Constructor Id="3" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}" /></ObjectPaths></Request>`
+        };
+
+        return request.post(requestOptions);
+      })
+      .then((res: string): void => {
+        const json: ClientSvcResponse = JSON.parse(res);
+        const response: ClientSvcResponseContents = json[0];
+
+        if (response.ErrorInfo) {
+          cb(new CommandError(response.ErrorInfo.ErrorMessage));
+          return;
+        }
+        else {
+          const orgAssetsResponse: OrgAssetsResponse = json[json.length - 1];
+
+          if (orgAssetsResponse.OrgAssetsLibraries === undefined) {
+            cb(new CommandError("No libraries in Organization Assets"));
```

Comment:
Should we log error or just output? I can see the powershell command is logging it as text
![image](https://user-images.githubusercontent.com/1049476/63147998-c4238400-bff7-11e9-9616-1a096b746fd3.png)